src/core/tasks.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import absolute_import
+import csv
+import ftplib
+import os
+
+from celery import task
```

Comment:
Django is the framework we use for this product, Celery is an external package. Nice to have:

```
from django.conf import settings

from celery import task
```